darts/tests/test_timeseries.py

```diff
@@ -975,6 +975,47 @@ def test_ops(self):
             # Cannot divide by 0.
             self.series1 / 0
 
+    def test_ops_samples_broadcasting(self):
```

Comment:
Cool test 👍 

I would maybe check the operations using a non-constant seriesB, so we can make sure that the broadcasting is performed properly along all axis/dimensions. What do you think?

Reply:
Can do. I started with that but it's much more work (and my implementation would just always fail because #2474 😅 ) so I thought I'd do a simpler one first.

Reply:
I think you can also just create a new `TimeSeries` within the test, if the pre-defined examples are not suitable for this test.

- I would generate the expected results using numpy arrays and operations since we can assume that numpy does the broadcasting as expected. 
- Then create a TimeSeries from the expected numpy arrays (with the time index and columns from the source/left series) and check that the TimeSeries broadcasting operations give the same result as these expected TimeSeries

Reply:
Added a more extensive test. Also found another bug, you could not divide TimeSeries by xarray or ndarray. I added another test for that and fixed it.